### PR TITLE
Update gateway.yml

### PR DESCRIPTION
**Pull request by nik-suri**

### CODE REVIEW

File: .github/DISCUSSION_TEMPLATE/gateway.yml [hunk 1]
```diff
@@ -60,96 +54,61 @@ body:
...
-       "# RPC"
-  - type: textarea 
-    id: rpc 
-    attributes:
-      label: RPC 
-      description: "Describe which dedicated RPCs are for both mainnet and testnet are available"
-      value: |
-        Wormhole marquee UIs such as [Portal Bridge](https://portalbridge.com/) and [Connect](https://wormhole.com/connect/) require dedicated RPCs to provide the best user experience. We ask that you coordinate with an RPC provider to set these up for the Wormhole UIs to use.
-    validations:
-      required: true
```
Comment (barnjamin) [inline .github/DISCUSSION_TEMPLATE/gateway.yml:155]: no need for this anymore?

File: .github/DISCUSSION_TEMPLATE/gateway.yml [hunk 2]
```diff
@@ -1,10 +1,6 @@
 title: "[Cosmos chain onboarding proposal] "
 labels: ["gateway"]
 body:
-  - type: markdown
-    attributes:
-      value: |
-       "# Overview"
```
Comment (barnjamin) [inline .github/DISCUSSION_TEMPLATE/gateway.yml:7]: Removing these also means the sections wont be broken up with headers anymore, intentional?

Comment (nik-suri) [reply .github/DISCUSSION_TEMPLATE/gateway.yml:7]: The headers already don't show up properly, so my opinion is they aren't of much use anyways.

**Review by barnjamin — APPROVED** [review]

File: .github/DISCUSSION_TEMPLATE/gateway.yml (hunk 1 above)
Comment (evan-gray) [reply .github/DISCUSSION_TEMPLATE/gateway.yml:155]: seems like they should be available at https://github.com/cosmos/chain-registry

**Review by evan-gray — APPROVED** [review]

**Pull request merged**